### DISCUSSION TITLE
Oven fully loaded

### DISCUSSION THREAD

**Discussion by ErneD** [frePPLe/frepple#606] [Q&A] [answered]
Hi Johan,

When modeling an oven it is better to fully load the resource in order to cook full batches, but Frepple is smoothing the load in order to avoid the resource to be loaded at 100%. This gives an unreal result because a batch that could be cooked using the oven 2 days at 100%  gets  cooked in 4 days or more. Do you know how could I prevent Frepple from smoothing the load over several days and load the oven at 100 % ?

Thank you !

Ernesto.

**Comment by jdetaeye** [comment, marked as answer]
If a resource isn't a critical bottleneck, it's normal that there will be some periods where it's not utilized at 100%.    

In its *backward planning* mode, Frepple generate a plan that meets the demand on a JIT-basis. You'll find empty periods in the schedule and periods where it's loaded less than 100% .   It's the planners job then to decide the grouping the manufacturing orders in 100%-loaded batches and idle periods.   This is manual, or could be automated with some customization.

In its *forward planning* mode, frepple creates a plan that schedules all demand ASAP - even before the sales order due date.  In such a plan you should see the ovens loaded at their maximum from the current date till all demand is planned.    Setting the parameter "plan.administrativeleadtime" to 999 will switch the solver to this mode.

See https://frepple.com/blog/backward-or-forward-scheduling/ for some more background.

**Reply by ErneD** [reply]
Hi Johan, thank you for your answer.
When you say automated with customization, you mean in Frepple ?
On the other hand, where can I set the parameter plan.administrativeleadtime ?

**Comment by jdetaeye** [comment]
>> When you say automated with customization, you mean in Frepple ?

Yes, frepple has a modular structure that allows apps/addons to be developed that customize the application with extra fields, extra reports, new styling, new planning logic, etc
Documentation for developers is available at https://frepple.com/docs/current/developer-guide/creating-an-extension-app.php   Such apps are can be developed by your team, or by the frepple team.

>> On the other hand, where can I set the parameter plan.administrativeleadtime ?

You find it in the admin/parameter screen.
https://frepple.com/docs/current/model-reference/parameters.php?highlight=administrativeleadtime

**Reply by ErneD** [reply]
Thank you !
